tools/config-generator/utils_test.go

```diff
@@ -331,3 +332,23 @@ func TestStrExists(t *testing.T) {
 		t.Fatalf("String bar should exist in %v", sArray)
 	}
 }
+
+func TestStringSetToSlice(t *testing.T) {
+	SetupForTesting()
+	strSet := map[string]struct{}{
+		"Harry":    {},
+		"Ron":      {},
+		"Hermione": {},
+	}
+
+	output := stringSetToSlice(strSet)
+	if len(output) != 3 {
+		t.Error("Expected len 3, was " + strconv.Itoa(len(output)))
```

Comment:
Nit: you can write it as
```suggestion
		t.Error("Expected len 3, was ", len(output))
```

Reply:
Done, thanks.